clang/tools/clang-format/git-clang-format

```diff
@@ -153,7 +157,10 @@ def main():
   else:
     if len(commits) > 2:
       die('at most two commits allowed; %d given' % len(commits))
-  changed_lines = compute_diff_and_extract_lines(commits, files, opts.staged)
+  changed_lines = compute_diff_and_extract_lines(commits,
```

Comment:
Maybe we can add a `die` when diff_from_common_commit is true and len(commits) is < 2